src/main/resources/application.properties

```diff
@@ -84,3 +84,6 @@ quarkus.log.cloudwatch.log-stream-name=policies-ui-backend
 quarkus.log.cloudwatch.level=INFO
 quarkus.log.cloudwatch.access-key-id=placeholder
 quarkus.log.cloudwatch.access-key-secret=placeholder
+
+# Environment
+env.name=""
```

Comment:
not sure if it's convention.. should we add a "quarkus." as prefix?

Reply:
I'm trying to load the already existent `ENV_NAME`:
https://quarkus.io/guides/config-reference#environment-variables

Reply:
No, `quarkus.` is supposed to be a reserved prefix used only for things coming from Quarkus.

Reply:
ok. was confused as we have that prefix for the cloudwatch extension as wll.